assets/scss/_search-results.scss

```diff
@@ -2,27 +2,27 @@
 @use 'variables';
 
 @media only screen and (max-width: 992px) {
-  .search__results-heading {
+  .search-results-heading {
```

Comment:
We should also keep the bem structure for class names, eg `search-results__heading` (this is what I mean by prefix).